docs/05-internals/design/api-v2/json-ld.md

```diff
@@ -19,8 +19,6 @@ License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
 
 # JSON-LD Parsing and Formatting
```

Comment:
Can you fix the code snippets? For example, in this file, there are several `@@snip` directives, which refer to snippets that are marked with special `// #` comments in the source code (e.g. in `ValuesRouteV2.scala`. These comments need to be deleted, and the snippets need to be copied and pasted into the documentation.

You can find all the Markdown files containing code snippets by searching for `snip` on docs.knora.org. :)

Reply:
ok, I've copied everything. no `@@snip` left.